samples/CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ if(GST_FOUND)
     GstAudioVideoReceiver.c
     kvsWebRTCClientMasterGstSample.c
   )
-  target_link_libraries(kvsWebrtcClientMasterGstSample kvsWebrtcClient kvsWebrtcSignalingClient ${EXTRA_DEPS} ${GST_SAMPLE_LIBRARIES} kvsCommonLws kvspicUtils websockets kvssdp kvsstun kvsrtp)
+  target_link_libraries(kvsWebrtcClientMasterGstSample kvsWebrtcClient kvsWebrtcSignalingClient ${EXTRA_DEPS} ${GST_SAMPLE_LIBRARIES} kvsCommonLws kvspicUtils)
```

Comment:
Do we need to remove `websockets kvssdp kvsstun kvsrtp` from [here](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/2009/files#diff-7d5e7fff524477259e472bf6c42b0a26419b75cc359bbb689712ec10641e32b9R64) as well?

Reply:
Good catch! Will fix it.